Week3/online_shop.sql

```diff
@@ -0,0 +1,31 @@
+CREATE DATABASE online_shop;
+
+,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+
+CREATE TABLE users(
+username varchar(50) NOT NULL,
```

Comment:
Is there a reason to have the 'username', in addition to the 'email', if the username isn't unique/distinct (since the email is the PK)?